src/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'redux/categories/CHECK_STATUS';
+
+const reducer = (state = [], action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under construction';
+    default:
+      return state;
+  }
+};
+
+export const checkStatus = () => ({ type: CHECK_STATUS });
```

Comment:
- [ ] [Optional] Same here, Please use `createSlice` from the `redux-toolkit` library to write your reducer.